frontend/app-development/features/administration/components/ResetRepoModal.tsx

```diff
@@ -30,14 +30,22 @@ export function ResetRepoModal(props: IResetRepoModalProps) {
 
   const onDeleteRepoNameChange = (event: any) => setDeleteRepoName(event.target.value);
 
-  const onResetWrapper = () => {
+  const repoResetMutation = useResetRepositoryMutation(org, app);
+  const onResetWrapper = async () => {
     setCanDelete(false);
-    props.handleClickResetRepo();
-    onCloseWrapper();
+    await repoResetMutation.mutateAsync();
+    setResetComplete(true);
```

Comment:
Trenger du egentlig denne `resetComplete`? Jeg tror det skal være tilstrekkelig å bruke `isSuccess` e.l. fra `repoResetMutation`. Da kan du også bruke `mutate` i stedet for `mutateAsync`.

Reply:
Oppdatert med `isSuccess` nå 😊